install/kots/manifests/gitpod-installer-job.yaml

```diff
@@ -273,6 +272,26 @@ spec:
                 'del(select(.kind == "StatefulSet" and .metadata.name == "openvsx-proxy").status)' \
                 "${GITPOD_OBJECTS}/templates/gitpod.yaml"
 
+              # Merge different docker pull secrets into one secret
+              if [ '{{repl HasLocalRegistry }}' = "true" ];
+              then
+                kubectl get secret "{{repl ImagePullSecretName }}" -o=jsonpath="{.data['\.dockerconfigjson']}" | base64 --decode | yq -P - > registry-auth-airgap.yaml
```

Comment:
`base64 --decode` should be `base64 -d` as --decode fails in busybox (fixed this in my test install)